packages/dask-task-models-library/src/dask_task_models_library/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.1.0"
```

Comment:
for your questions about version. I would vote for VERSION as the one and only source of truth.